docs/articles/new-expensify/settings/Switch-to-light-or-dark-mode.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Switch to light or dark mode
+description: Change the appearance of Expensify
+---
+<div id="new-expensify" markdown="1">
+
+Expensify has three themes that determine how the app looks:
+- **Dark mode**: The app appears with a black background
+- **Light mode**: The app appears with a white background
+- **Use Device settings**: Expensify will automatically use your device’s default theme 
```

Comment:
```suggestion
Expensify has three theme options that determine how the app looks:
- **Dark mode**: The app appears with a dark background
- **Light mode**: The app appears with a light background
- **Use Device settings**: Expensify will automatically use your device’s default theme 
```

Nitpicks, but it's not really _three themes_ or white/black... let me know what you think though!

Reply:
MUCH BETTER! Thank you!